_stbt/core.py

```diff
@@ -782,7 +823,12 @@ def detect_motion(self, timeout_secs=10, noise_threshold=None, mask=None):
             imglog.imwrite("absdiff_threshold", thresholded)
             imglog.imwrite("absdiff_threshold_erode", eroded)
 
-            motion = (cv2.countNonZero(eroded) > 0)
+            out_region = _nonzero_region(eroded)
+            if out_region:
+                # Undo cv2.erode above:
+                out_region = out_region.extend(x=-3, y=-3, bottom=3, right=3)
```

Comment:
Yes, good thinking.  Here's what full-screen motion looks like:

![video 2](https://cloud.githubusercontent.com/assets/494659/16037134/15e3dbae-3217-11e6-8827-64a65dc5bf09.gif)

So the highlight is still there but the label disappears off the top of the screen.